README.md

```diff
@@ -1,2 +1,24 @@
 # bot
 Custom bot, built in discord.js, for use with NGS' Discord server.
+
+## Getting Started
+
```

Comment:
I'd love to use `nvm` here and lock the project to a specific version of `npm` and `node`.

Reply:
I have added node to the dependencies of the package - what will require having said node version and will cause npm to install that when running the install package; (however user will need node & npm (come together) in order to get a toe in the water)
From what I can tell nvm is just an environment manager to switch between node versions, am I missing something - I don't have nvm installed.

Reply:
Yup, that's exactly right - `nvm` ensures that every single person running on this project is running the same version of `node` and `npm`. Without control there, you're going to run into situations where `npm install` might generate noise / garbage in `package.json` that varies from person to person, and now PRs become polluted.

Super easy to install - I'd pick the latest and greatest.

We can also consider using `yarn` over `npm`.

Reply:
Well I've been mulling it over today and I'm still a bit skeptical about the gains of doing this --
From what I can tell the nvm is a great tool especially for mac/linux users but from what i've gathered the nvmrc file doesn't work with the windows style nvm, it seems like it's a nice utility tool for folks who want to use nvm (sort of like a rb-env) but forcing it into the project doesn't seem like the right play to me.  
Further if we want to use yarn in leu or along side of npm then that should be fine I've seen lots of repos using both but I don't see any clear reason do one instead of the other
In short I don't fully see the need or gain from implementing another layer of complexity to the repo as lots of repos run with only node and npm, the package and package-lock handle the dependency tracking and it should only grow when we add new dependencies

Reply:
All fine - if we run into any issues with package installation or `npm` specific changes to `package.json` due to specific versions (which is what this tries to address), we can revisit this later.